docs/develop/scripting-fundamentals.md

```diff
@@ -11,20 +11,25 @@ This article will introduce you to the technical aspects of Office Scripts. You'
 
 [!INCLUDE [Preview note](../includes/preview-note.md)]
 
-## `main` function
+## TypeScript: The language of Office Scripts
 
-Each Office Script must contain a `main` function with the `ExcelScript.Workbook` type as its first parameter. When the function is executed, the Excel application invokes this `main` function by providing the workbook as its first parameter. Hence, it is important to not modify the basic signature of the `main` function once you have either recorded the script or created a new script from the code editor.
+Office Scripts are written in [TypeScript](https://www.typescriptlang.org/docs/home.html), which is a superset of [JavaScript](https://developer.mozilla.org/docs/Web/JavaScript). If you are familiar with JavaScript, your knowledge will carry over (much of your code is the same in both languages). We recommend you have some beginner-level programming knowledge before starting your Office Scripts coding journey. The following resources can help you understand the coding-side of Office Scripts.
+
+[!INCLUDE [Preview note](../includes/coding-basics-references.md)]
+
+## `main` function: The script's starting point
+
+Each Office Script must contain a `main` function with the `ExcelScript.Workbook` type as its first parameter. When the function is executed, the Excel application invokes this `main` function by providing the workbook as its first parameter. An `ExcelScript.Workbook` should always be the first parameter.
```

Comment:
"Avoid execute except to follow the UI. Use run instead. If the UI includes execute, the resulting action is still run." - From the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33512). 
```suggestion
Each Office Script must contain a `main` function with the `ExcelScript.Workbook` type as its first parameter. When the function runs, the Excel application invokes this `main` function by providing the workbook as its first parameter. An `ExcelScript.Workbook` should always be the first parameter.
```